src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.exception.handlers.FruitTransactionException;
+import org.junit.Test;
+
+public class FruitTransactionTest {
```

Comment:
Actually we don't recommend testing this model in terms of this task, as it's not related to services' business logic